### DISCUSSION TITLE
Graph library

### DISCUSSION THREAD

**Discussion by ArmborstL** [utwente-fmt/vercors#1139] [Wishlist]
We have had multiple case studies relating to graphs, such as the Ford-Fulkerson algorithm and Kahn's topological sorting algorithm. Each defined their own graph data structure. It would be nice to have a (verified) graph library implementing the general structure, as well as functions like `getSuccessors`, to simplify getting started on new graph algorithms, and unify the case studies.

**Comment by bobismijnnaam** [comment]
I would also like to mention Han & Jan's ([1], [2]) masterstheses for this list.

[2]: https://purl.utwente.nl/essays/94474
[1]: https://purl.utwente.nl/essays/88268
